src/app/pages/back-to-office/back-to-office.page.html

```diff
@@ -1,10 +1,22 @@
 <ion-header>
-  <ion-toolbar>
-    <ion-title>back-to-office</ion-title>
-  </ion-toolbar>
+    <practice-mode-banner *ngIf="isEndToEndPracticeMode"></practice-mode-banner>
 </ion-header>
 
-<ion-content>
-  <ion-button (click)="navigateBack()">Back</ion-button>
-  <ion-button (click)="navigateForward()">To Office</ion-button>
+<ion-content id="post-debrief-holding-cat-b-page" class="background-colour">
```

Comment:
Update id to be relevant to page, DES3 = 'back-to-office-page'. 

Also, in DES3 there is a no-bounce attribute on the ion-content, this has been replaced by 'force-overscroll="false"'

DES3 implementation - https://github.com/dvsa/mes-mobile-app/blob/develop/src/pages/back-to-office/cat-b/back-to-office.cat-b.page.html